server/media/media_api.py

```diff
@@ -89,6 +89,9 @@ def __handle_file_upload():
     else:
         return "Unsupported file format", 415
 
+    result.title = request.form.get("title")
```

Comment:
what happens if the field is missing? Does get() throw some errors?

Reply:
Returns `default`-value, and by default `default` is `None` 😄